samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -219,6 +219,14 @@ static constexpr char inference_only_message[] =
     " To enable full mode for static models pass \"false\" value to this argument:"
     " ex. \"-inference_only=false\".\n";
 
+static const char frames_per_second[] =
+     "Optional. Gives possibility to configure the maximum workload processing speed in frames per second (FPS) metric.\n" 
+     "Based on that number application calculates target latency for each data frame.\n"
+     "If the VPU process actual frame faster than the target latency it waits for new frame the time:\n"
+     "target latency -<actual_frame_processing_time>."
+     "If not specified it will be calculated as a derivative from:"
+     "reported latency in the Sync mode and the total execution time in the Async mode.";
+
```

Comment:
General question why we need this feature in benchmark_app? benchmark_app is a tool which measures FPS, so FPS is output of this tool, not input parameter. 

Whether it is VPU specific?